internal/app/machined/pkg/controllers/runtime/kernel_param_defaults.go

```diff
@@ -120,6 +121,11 @@ func (ctrl *KernelParamDefaultsController) getKernelParams() []*kernel.Param {
 			Key:   "proc.sys.net.ipv4.tcp_keepalive_intvl",
 			Value: "60",
 		},
+		// Consider fwmark for rp_filter routing table lookup.
+		{
+			Key:   "proc.sys.net.ipv4.conf." + constants.KubeSpanLinkName + ".src_valid_mark",
```

Comment:
this probably doesn't apply if the KubeSpan is not enabled, should we instead create a resource with this sysctl value from the KubeSpan manager controller, so it gets created/removed with KubeSpan enabled/disabled?

Reply:
Perhaps. I also thought there might be issues if this is applied before KubeSpan interface is up something might not work. Will fix